src/extension/wps/wps-core/src/main/java/org/geoserver/wps/gs/ImportProcess.java

```diff
@@ -280,8 +282,8 @@ else if (coverage != null) {
         else if (coverage != null)
         {
         	try {
-        		final File directory = catalog.getResourceLoader().findOrCreateDirectory("data", workspace, store);
-        		final File file = File.createTempFile(store, ".tif", directory);
+        		final Resource directory = catalog.getResourceLoader().get(Paths.path("data", workspace, store));
+        		final File file = File.createTempFile(store, ".tif", directory.dir());
```

Comment:
Yes, we could, we probably should too... it would take a champion, and a bit of coordination: all the active branches should be formatted at once, with minimal pull requests open, and with no big works in progress in forks while this happens. 
I guess the Christmas vacations could be a good time to do this.